zulip_bots/zulip_bots/bots/xkcd/xkcd.py

```diff
@@ -49,7 +49,7 @@ class XkcdServerError(Exception):
 
 def get_xkcd_bot_response(message):
     original_content = message['content'].strip()
-    command = original_content.strip()
+    command = original_content.split(' ', 1)[-1].strip()
```

Comment:
The `.strip()` at this line might be redundant with the previous one.

Reply:
Technically not since I'm splitting on a single white space and there might be several in between the bottom name and argument. But this is not robust in any case :-) this is probably worth handling in a library that multiple Bots could use.

Reply:
The logic (removing away the bot @-mention), LGTM.

Reply:
> ... and there might be several in between the bottom name and argument
Yeah, I see this can be handled via `.split()` (without an explicit `' '`) or regexp split [1]

[1] https://stackoverflow.com/questions/13028120